source.list

```diff
@@ -87,7 +87,15 @@ opendune.c
 os/endian.c
 #if WIN32
 	os/error_win32.c
-	os/thread_win32.c
+	#if SDL
+		os/thread_sdl.c
+	#else
+		#if SDL2
+			os/thread_sdl.c
+		#else
+			os/thread_win32.c
+		#endif
+	#endif
```

Comment:
Added directives to create correctly MSVC Project files (.vcxproj), already activating SDL1.2/SDL2.0 files on project into solution